docs/source/platforms/radxa/rock4.rst

```diff
@@ -0,0 +1,36 @@
+.. |yes| image:: ../../images/yes.png
+.. |no| image:: ../../images/no.png
+
+.. role:: underline
+   :class: underline
+
+rock4
+=====================
+
++----------------+-----------------+
+| System on Chip | Broadcom 2836   |
++----------------+-----------------+
+| Number GPIO    | 27              |
++----------------+-----------------+
+| Status         |                 |
```

Comment:
Can you fill in the `Status`

Reply:
Thank you for your guidance，and I'm sorry to bother you for my careless .